src/script.ts

```diff
@@ -113,6 +113,17 @@ function enableNightMode(): void {
     document.body.classList.add("nightmode");
 }
 
+function updateClock(): void {
+    const m = new Date();
+    const clockTextElm = document.getElementById("clock-text");
+    if (clockTextElm == null) {
+        console.log("clock text element not found");
+    } else {
+        clockTextElm.textContent = (m.getHours() < 10 ? ("0" + m.getHours().toString()) : m.getHours())
+             + ":" + (m.getMinutes() < 10 ? ("0" + m.getMinutes().toString()) : m.getMinutes());
```

Comment:
what a shame JS doesn't have basic strftime support. in any case, prefer Freedom Units© because This Is America™

![](https://www.meme-arsenal.com/memes/5db911d12be3094c1bf1b7730be3c4a1.jpg)

Reply:
Isn't this already in customary units or do you mean the AM/PM stuff?